azkaban-common/src/test/resources/azkaban/executor/embedded/innerFlow.job

```diff
@@ -0,0 +1,5 @@
+type=javaprocess
+java.class=azkaban.test.executor.SleepJavaJob
```

Comment:
Thanks! Actually, something like that is not a bad idea. I have removed the duplicated test files and added a utility class to `azkaban-test` (`TestExecutions`) with a method that other tests can use to get a `File` object for the directory of one of the flows in `azkaban-test`.

This way, we build a JAR from `azkaban-test` that contains this class and all the test files, and other tests can use the `TestExecutions` class, which will load the resource files in the `azkaban-test` jar using its own class loader.

I have also moved all the test flows in `azkaban-test` into the correct resource directory so that they can be loaded from the jar, though this would mean that we would need to update the paths in the `TestUtils` class that you wrote.